src/main/java/uk/gov/hmcts/reform/ccd/document/am/controller/endpoints/CaseDocumentAmController.java

```diff
@@ -129,24 +120,18 @@ public CaseDocumentAmController(DocumentManagementService documentManagementServ
         consumes = {APPLICATION_JSON})
     public ResponseEntity<Object> patchDocumentbyDocumentId(
         @ApiParam(value = "", required = true)
-        @Valid UpdateDocumentCommand body,
+        @Valid @RequestBody UpdateDocumentCommand body,
         @PathVariable("documentId") UUID documentId) {
-        ResponseEntity responseEntity = documentManagementService.getDocumentMetadata(documentId);
-        if (documentManagementService.checkUserPermission(responseEntity, documentId, Permission.UPDATE)) {
-            ResponseEntity response =   documentManagementService.patchDocument(documentId, body);
-            return  ResponseEntity.status(HttpStatus.OK).body(response.getBody());
-        }
-        LOG.error("User doesn't have update permission on requested document {}", HttpStatus.FORBIDDEN);
-        throw new ForbiddenException(documentId.toString());
+        ResponseEntity response =   documentManagementService.patchDocument(documentId, body);
```

Comment:
We don't need to check permission before update the patch.

Reply:
Yes, same as above.